dev/drivers/scripts/mesoscale/stats/jevs_mesoscale_nam_grid2obs_stats.sh

```diff
@@ -63,8 +63,9 @@ set -x
 
```

Comment:
Hi, Shelley,

Just a note:  the code lines 49 and 50 wouldn't work in the emc.vpppg parallel.  That word "verification" should have been removed.  Of course, no one is looking at the parallel as instructed.  

Please check these mesoscale drivers and make corrections.

Perry

Reply:
Thankfully, the emc.vpppg parallel somehow doesn't include this line - it sets HOMEevs properly.  But still they should be in the repository properly.

Perry

Reply:
Sure! I can do an overall `export HOMEevs=` check and correct where needed. I know there are a few other offending components.